app/src/main/java/pl/szczodrzynski/edziennik/App.kt

```diff
@@ -58,6 +58,7 @@ class App : MultiDexApplication(), Configuration.Provider, CoroutineScope {
         val profileId
             get() = profile.id
 
+        var chucker = false
```

Comment:
Change this to `enableChucker` here and in Config.

Reply:
done